storage/StorageBlockLayout.proto

```diff
@@ -83,6 +84,22 @@ message SMAIndexSubBlockDescription {
   }
 }
 
+message BloomFilterIndexSubBlockDescription {
+  extend IndexSubBlockDescription {
+    repeated int32 indexed_attribute_id = 33;
+
+    // The default values are added for a 2 MB block size, which were
+    // determined from empirical experiments. These values control the
+    // amount of false positivity that is expected from Bloom Filter.
+    // - Default bloom filter size per attribute = 10 KB.
+    // - Default number of hash functions used in bloom filter = 20.
+    // - Default number of elements expected to be stored = 1 million.
+    optional fixed64 bloom_filter_size = 34 [default = 10000];
+    optional int32 number_of_hashes = 35 [default = 20];
+    optional fixed64 projected_element_count = 36 [default = 1000000];
```

Comment:
These numbers (from 33 to 36) assigned to the fields does not follow the rules described in `IndexSubBlockDescription`.